### ISSUE TITLE
preLaunchTask don't start unless multiple click on Start/Restart button 

### ISSUE THREAD

**Issue by mic4126** [microsoft/vscode#73590]
Issue Type: <b>Bug</b>

similar to #72950

Step to Repeat problem:
Step 1: start debug chrome
Step 2: close the chrome
Step 3: start debug chrome again

Step 3 will not be able to recompile the js 

VS Code version: Code - Insiders 1.34.20-insider (d6aa8eaf30c71df32d4543290675808440def630, 2019-05-09T22:10:02.019Z)
OS version: Windows_NT x64 10.0.16299

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 7 2700X Eight-Core Processor          (16 x 3700)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.95GB (20.61GB free)|
|Process Argv||
|Screen Reader|no|
|VM|50%|
</details><details><summary>Extensions (2)</summary>

Extension|Author (truncated)|Version
---|---|---
Go|ms-|0.10.2
debugger-for-chrome|msj|4.11.3

</details>

stripped from my working repo but still could reproduce the problem.
[prelaunch-bug.zip](https://github.com/microsoft/vscode/files/3167862/prelaunch-bug.zip)

**Comment by mic4126** [comment]
After submitted, i also tried the version commented below
版本: 1.34.20-insider (system setup)
認可: d6aa8eaf30c71df32d4543290675808440def630
日期: 2019-05-09T22:10:02.019Z
Electron: 3.1.8
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.16299

**Comment by isidorn** [comment]
Stoping debugging does not stop the prelaunch task.
If you manually terminate the task after step 2 with F1 > Tasks: Terminate Task does step 3 work ok then?

When opening your repository you shared the `webpack_dev` task can not be found. Due to that assigning also to @alexr00 to investigate for tasks

**Comment by alexr00** [comment]
@isidorn I can't reproduce the `webpack_dev` task not found. I can run the task without problems.
I also cannot reproduce the issue. I see that the web pack task is run ever time that the launch config is run.

Details:
Version: 1.35.0-insider (user setup)
Commit: 16de3c28db1d2c6fbe2227d1ffe91c5e7e683bc9
Date: 2019-05-27T05:19:20.616Z
Electron: 3.1.8
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17763

@mic4126 can you try with the latest insiders? Do you have any additional repro steps?

**Comment by mic4126** [comment]
Sorry for the late reply.
I have just tried with the latest insiders version that still saying task not found,
(版本: 1.35.0-insider (system setup)
認可: 16de3c28db1d2c6fbe2227d1ffe91c5e7e683bc9
日期: 2019-05-27T05:19:20.616Z
Electron: 3.1.8
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.16299)

**Comment by mic4126** [comment]
the zip include the screep rec video about how to reproduce the bug on my pc
[2019-05-27 19-13-30.zip](https://github.com/microsoft/vscode/files/3223505/2019-05-27.19-13-30.zip)

**Comment by alexr00** [comment]
I have tried doing the same thing as you show in the video, but I still see do not see the bug that you see.
Can you try adding args like this:
```
    "windows": {
            "command": "${workspaceRoot}\\node_modules\\.bin\\webpack.cmd",
            "options": {
                "shell": {
                    "executable": "${env:COMSPEC}",
                    "args": ["/C"]
                },
                "cwd": "${workspaceFolder}"
            }
        },

**Comment by mic4126** [comment]
tried to add `"/C"`
still same result.

I remember in the previous issue #72950. I install the insider version, then the problem disappear. 
The problem only occur after using the insider version for a period. 

so is there anyway i can try to reset vscode and try to reproduce the problem again.

**Comment by alexr00** [comment]
@mic4126 can you paste in your settings.json? Next time you see the issue, can you also open the developer tools (Developer: Toggle Developer Tools command) and see if there is anything in the console?

**Comment by mic4126** [comment]
settings.json in Addata

[settings.zip](https://github.com/microsoft/vscode/files/3248667/settings.zip)

settings inside .vscode
[settings.zip](https://github.com/microsoft/vscode/files/3248669/settings.zip)

**Comment by mic4126** [comment]
![image](https://user-images.githubusercontent.com/4671533/58820578-899b8380-8665-11e9-8fea-00c51052b918.png)
dev tools  -> console

**Comment by alexr00** [comment]
@mic4126 is this still happening?

**Comment by mic4126** [comment]
yes

**Comment by mic4126** [comment]
Version: 1.39.0-insider (system setup)
Commit: 313ede61cbad8f9dc748907b3384e059ddddb79a
Date: 2019-10-01T07:29:05.652Z
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.16299

**Comment by alexr00** [comment]
The task in your sample would not work because it is missing the `/c` from https://github.com/microsoft/vscode/issues/73590#issuecomment-496324669

Can you share your new task with the `/c`?
